### PR TITLE
minor cleanup

### PR DESCRIPTION
**Pull request by Sicprofundus**
- cleanup PSPAWNINFO casts / usage

### CODE REVIEW

File: MQ2ItemScore.cpp [hunk 1]
```diff
@@ -794,6 +794,7 @@ void SetReportChannel(char* Val)
 
 int SetClickMode(int Val, char* Opt)
 {
+	// TODO:: Check validity of the !Val return
```
Comment (brainiac) [inline MQ2ItemScore.cpp:797]: i don't think this function is even used.

File: MQ2ItemScore.cpp [hunk 2]
```diff
@@ -806,6 +807,7 @@ int SetClickMode(int Val, char* Opt)
 void SetClickMode(char* Val, char* Opt)
 {
 	if (_stricmp(Val, "Channel") == 0) strcpy_s(ClickChannel, Opt);
+	// TODO: Check the Validity of the !Clicky___
```
Comment (brainiac) [inline MQ2ItemScore.cpp:810]: what validity do you want to check? These look like a toggle.

Comment (Sicprofundus) [reply MQ2ItemScore.cpp:810]: Just that it is valid usage/intent. Visual studio says ~ was probably intended, I'm guessing just "0" was intended, but wanted to come back to that

Comment (brainiac) [reply MQ2ItemScore.cpp:810]: Apart from the fact that these variables should've been written as `bool`, theres nothing inherently wrong with the logic here.

**Comment by Sicprofundus** [comment]
closing MR after conversation with brainiac.

**Pull request closed without merging**